### ISSUE TITLE
pytest_cov.__version__ string is broken on >=2.10.1

### ISSUE THREAD

**Issue by christophebedard** [pytest-dev/pytest-cov#468]
# Summary

## Expected vs actual result

I expect `pytest_cov.__version__ == 'x.y.z'`, but as of 2.10.1 it's `pytest_cov.__version__ == "__version__ = 'x.y.z'"`: https://github.com/pytest-dev/pytest-cov/commit/da61b912c48fadf6df71c8cd4712a23dbdc2efc0#diff-cfb899e3f5cac1afba8f5bafcd74392faf2677836ed2f3b068217df610ea6e82R2

This broke code that relied on the version string. There's no explanation in the CHANGELOG: https://github.com/pytest-dev/pytest-cov/blob/2cd009442ddb08dc873117019b13ef972803d5c4/CHANGELOG.rst#L36-L42

Looking at PEP 396 I'm not sure this is valid: https://www.python.org/dev/peps/pep-0396/#id27

# Reproducer

## Versions

Output of relevant packages `pip list`, `python --version`, `pytest --version` etc.

```
$ python3 --version
Python 3.8.5
$ pytest --version
pytest 6.2.2
$ pytest --version --version
This is pytest version 6.2.2, imported from /home/chris/.local/lib/python3.8/site-packages/pytest/__init__.py
setuptools registered plugins:
  pytest-rerunfailures-9.1.1 at /home/chris/.local/lib/python3.8/site-packages/pytest_rerunfailures.py
  pytest-repeat-0.9.1 at /home/chris/.local/lib/python3.8/site-packages/pytest_repeat.py
  pytest-cov-2.11.1 at /home/chris/.local/lib/python3.8/site-packages/pytest_cov/plugin.py
  colcon-core-0.6.1 at /usr/lib/python3/dist-packages/colcon_core/pytest/hooks.py
  pytest-mock-1.10.4 at /usr/lib/python3/dist-packages/pytest_mock.py
```

## Config

N/A

## Code

```py
>>> from pkg_resources import parse_version
>>> from pytest_cov import __version__ as pytest_cov_version
>>> pytest_cov_version
"__version__ = '2.11.1'"
>>> parse_version('2.11.1') >= parse_version('2.5.0')
True
>>> parse_version(pytest_cov_version) >= parse_version('2.5.0')
False
```

**Comment by christophebedard** [comment]
I think this might be caused by a tooling bug. A few commits before `2.10.1` was tagged, this config file here for bumping the version number was changed: https://github.com/pytest-dev/pytest-cov/pull/420/files#diff-9ea6e1e3dde6d4a7e08c7c88eceed69ca745d0d2c779f8f85219b22266efff7f (https://github.com/pytest-dev/pytest-cov/commit/c4b0722566c356fe2fd3820dcafba1f0d89d325f).

I'll try to fix it.

**Comment by ionelmc** [comment]
A unfortunate consequence of me using old bumpversion and messing up the quotes :-(

**Comment by christophebedard** [comment]
:laughing: thanks for fixing it!

**Comment by dirk-thomas** [comment]
Thanks for addressing this. Will there a new patch release with this fix in the near future?

**Comment by ionelmc** [comment]
Yes, yes. Gonna sort it out this week.

**Comment by ionelmc** [comment]
Just released 2.12
